<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_day</class>
 <widget class="QWidget" name="w_day">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>283</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>283</width>
    <height>40</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 50);</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>8</y>
     <width>85</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Понедельник</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>8</y>
     <width>60</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>+23 С</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>8</y>
     <width>135</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>135</width>
     <height>18</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>135</width>
     <height>18</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>солнечно</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
